# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/naga/Cargo.toml
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 third_party/rust/naga/Cargo.toml
--- a/third_party/rust/naga/Cargo.toml	Fri Oct 03 18:01:30 2025 +0000
+++ b/third_party/rust/naga/Cargo.toml	Fri Oct 03 18:08:29 2025 +0000
@@ -119,7 +119,7 @@
 default-features = false
 
 [dependencies.hashbrown]
-version = "0.15"
+version = "0.16"
 features = [
     "default-hasher",
     "inline-more",
@@ -131,7 +131,7 @@
 optional = true
 
 [dependencies.indexmap]
-version = "2.7"
+version = "2.8"
 default-features = false
 
 [dependencies.libm]
@@ -195,7 +195,7 @@
 default-features = false
 
 [dev-dependencies.hashbrown]
-version = "0.15"
+version = "0.16"
 features = [
     "default-hasher",
     "inline-more",
@@ -214,7 +214,7 @@
 path = "../naga-test"
 
 [dev-dependencies.ron]
-version = "0.10"
+version = "0.11"
 
 [dev-dependencies.rspirv]
 version = "0.12"
@@ -231,7 +231,7 @@
 version = "0.3"
 
 [dev-dependencies.strum]
-version = "0.27"
+version = "0.27.1"
 features = ["derive"]
 default-features = false
 